stats

```diff
@@ -9,7 +10,23 @@
  * file that was distributed with this source code.
  */
 
-require 'vendor/autoload.php';
+$loaded = false;
```

Comment:
actually, i've changed my mind - we want the composer variables to end up global, so this is fine as is :)